hack/externalnode/install-vm.ps1

```diff
@@ -167,41 +187,82 @@ function UpdateAgentConf() {
             Log "Updating $AntreaAgentConfPath with ${ExternalNodeNamespace}: ${Namespace}"
             [System.IO.File]::AppendAllText($AntreaAgentConfPath, "  ${ExternalNodeNamespace}: ${Namespace}" +
                     ([Environment]::NewLine))
+        } elseif ($line -like "*$Bridge*") {
+                Log "Updating $AntreaAgentConfPath with ${Bridge}: ${OVSBridge}"
+                [System.IO.File]::AppendAllText($AntreaAgentConfPath, "${Bridge}: ${OVSBridge}" +
+                        ([Environment]::NewLine))
         } else {
             [System.IO.File]::AppendAllText($AntreaAgentConfPath, $line +
                     ([Environment]::NewLine))
         }
     }
 }
 
+function ClearOVSConfig() {
+    Log "Deleting OVS bridge $OVSBridge"
+    Stop-Service $AntreaAgent
+    try {
+        $adapterName = (Get-VMNetworkAdapter -SwitchName $AntreaSwitch -ManagementOS).Name
```

Comment:
> Do you miss parameter `-ComputerName $(hostname.exe)` here which is used in remove-vmswitch command?

Will update it.

Reply:
> Is it possible that multiple VMNetworkAdapters existing on the VMSwitch in some cornor cases, or `$adapterName` is empty?

Assumptions here are that, 
- antrea-switch VMSwitch is created only by antrea-agent.
- we only support one interface currently. 

If the adapterName is empty, it implies that there is no antrea-switch. In that case, cleanup will fail with an error, which is fine. 
The cleanup service is for the correct scenarios, not meant to recover from other types of failure.

Reply:
With the assumption that only one network adapter returns, it only works after startup (reboot). So it may hit issues if antrea-cleaup is run if the Windows host is running (not a startup), as we have one uplink and one internal vnic, right?

Reply:
The antrea-cleanup service will only run once at startup. This is how the install logs will look like.
When the antrea-cleanup service is started as part of antrea installation, then there will be an error (since br-int bridge is not present in OVS) which is ignored. After the installation, the cleanup service will run only once after each startup.

10/11/2022 6:29 AM Checking supported Windows OS versions 
10/11/2022 6:29 AM Check Hyper-v feature is enabled 
10/11/2022 6:29 AM Check OVS services are installed 
10/11/2022 6:29 AM Copying C:\Users\nsxadmin\Desktop\antrea-agent\antrea-agent.exe to C:\antrea-agent 
10/11/2022 6:29 AM Copying C:\Users\nsxadmin\Desktop\antrea-agent\antrea-agent.kubeconfig to C:\antrea-agent\conf 
10/11/2022 6:29 AM Copying C:\Users\nsxadmin\Desktop\antrea-agent\antrea-agent.antrea.kubeconfig to C:\antrea-agent\conf 
10/11/2022 6:29 AM Updating C:\antrea-agent\conf\antrea-agent.conf with ovsBridge: br-int 
10/11/2022 6:29 AM Updating C:\antrea-agent\conf\antrea-agent.conf with externalNodeNamespace: vm-ns 
10/11/2022 6:29 AM Updating C:\antrea-agent\conf\antrea-agent.conf with kubeconfig: C:\antrea-agent\conf\antrea-agent.kubeconfig 
10/11/2022 6:29 AM Updating C:\antrea-agent\conf\antrea-agent.conf with kubeconfig: C:\antrea-agent\conf\antrea-agent.antrea.kubeconfig 
10/11/2022 6:29 AM Copying install script to C:\antrea-agent\conf 
10/11/2022 6:29 AM Creating service antrea-agent C:\antrea-agent\antrea-agent.exe --config C:\antrea-agent\conf\antrea-agent.conf --log_file C:\antrea-agent\logs\antrea-agent.log --logtostderr=false 
10/11/2022 6:29 AM Deleting OVS bridge br-int 
10/11/2022 6:29 AM Failed to get VMSwitch antrea-switch , rc Hyper-V was unable to find a virtual switch with name "antrea-switch". 
10/11/2022 6:29 AM Ignoring start error for antrea-cleanup service 
10/11/2022 6:31 AM Deleting OVS bridge br-int 
10/11/2022 6:31 AM Done Deleting OVS bridge br-int 
10/11/2022 6:33 AM Deleting OVS bridge br-int 
10/11/2022 6:33 AM Done Deleting OVS bridge br-int

Reply:
Yes, but I am asking if user manually run the service without a reboot...

Reply:
Case 1: If an external node is configured, and antrea-agent has created VmSwitch.
The cleanup routine will stop antrea-agent, cleanup the bridge and restore the interfaces and then start antrea-agent. Once antrea-agent starts it will again watch for externalnode and create antrea-switch again.

`Note: there will be network disconnectivity while remove and creating antrea-switch.`

Case 2:
If there is no externalnode configured for the VM, then this operation will be a no-op.

Reply:
Having offline confirmed that the uplink adapter is not shown in the result when listing the vmnetwork adatpers on the VMSwitch, even if it is already added on OVS bridge, so this concern is gone.